supabase/migrations/20260824144021_revoke_anon_oracle_rpc_execute.sql

```
@@ -0,0 +1,22 @@
+-- Revoke anonymous execute on SECURITY DEFINER RPCs that leak distinguishable
+-- outcomes (org/app/key existence) to unauthenticated PostgREST callers.
+-- Authenticated JWT and service_role callers keep access for console, CLI, and edge.
```

SQLFluff (4.3.0)

[error] Line is too long (85 > 80).

(LT05)